core/backend/src/TextAnnotationLayout.ts

```diff
@@ -41,18 +41,12 @@ export type FindFontId = (name: string) => FontId;
 /** @internal */
 export type FindTextStyle = (name: string) => TextStyleSettings;
 
-/** Arguments supplied to [[computeTextBlockExtents]].
- * @beta
- */
-export interface ComputeTextBlockExtentsArgs {
+/** @internal */
```

Comment:
do we need to call this out in nextver.md?

Reply:
It's a beta API that only existed for a few weeks and nobody ever used.